source/features/show-open-prs-of-forks.tsx

```diff
@@ -69,7 +72,7 @@ async function initHeadHint(): Promise<void | false> {
 
 async function initDeleteHint(): Promise<void | false> {
 	const [count, url] = await getPRs();
-	if (!count) {
+	if (!count || !location.pathname.endsWith('settings')) {
```

Comment:
This needs to be more precise (this also matches `/fregante/webext-settings`) and belongs to the `exclude`. It also already exists: https://github.com/fregante/github-url-detection/blob/875cf8a5e67bb1332290f52dc1f7a8ec0c50d3e1/index.ts#L299
```js

	exclude: [
		() => !pageDetect.isForkedRepo(),
		pageDetect.isRepoSettings
	],
```

Reply:
The error is when going to the setting, we only want to run it on the page the has the delete option.
In other words we are in the settings.

Correct me if I am wrong.

I moved it to exclude in f5f35edc1e185c364f5650bbe37ef9ec5968420b

Reply:
You can use https://github.com/fregante/github-url-detection/pull/2

Reply:
Done